spring-vault-core/src/main/java/org/springframework/vault/core/VaultTransitTemplate.java

```diff
@@ -174,6 +178,49 @@ public String encrypt(String keyName, byte[] plaintext,
 	}
 
 	@Override
+	public List<Map<String, String>> encrypt(String keyName, List<String> plaintexts) {
+
+		return encrypt(keyName, plaintexts, null);
+	}
+	
+	@Override
+	public List<Map<String, String>> encrypt(String keyName, List<String> plaintexts,
+			List<VaultTransitContext> transitRequests) {
+		
+		Boolean hasContext = false;
```

Comment:
How about using the primitive `boolean`? There's not need for three states here.